src/actions/BookActions.js

```diff
@@ -0,0 +1,28 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+const EDIT_BOOK = 'EDIT_BOOK';
+const UPDATE_PROGRESS = 'UPDATE_PROGRESS';
+
+export const addBook = book => ({
+  type: ADD_BOOK,
+  payload: book,
+});
+
+export const removeBook = bookID => ({
+  type: REMOVE_BOOK,
+  payload: bookID,
```

Comment:
- Take a book object as a parameter. You can then access the ID from the object itself. 
- Move all these action objects to `src/actions/index.js` file.

Reply:
- this is required in the book component not here.
- all actions are moved.